moon-store-mac-os/moon-store-mac-os/Helpers/Extensions/View+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  View+Extension.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/27/24.
+//
+
+import SwiftUI
+
+extension View {
+    func leadingInfinity() -> some View {
+        self.frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
Podes remover la suggar syntax 

```suggestion
        frame(maxWidth: .infinity, alignment: .leading)
```